docs/src/howto-guides/010_add_self_signed_trusted.md

```diff
@@ -0,0 +1,27 @@
+# How to add self-signed certificate to trusted
+
```

Comment:
It would be good to outline the reasons why someone might want to do this before just jumping into the instructions.

Reply:
@lines 5-7

Reply:
So why would someone want to `manually add a self-signed certificate to be trusted by Linux system`?

It should talk about self-signed certificates on the server, or certificates with a root that is not trusted. Tie it into someone trying to operate thin-edge.

Reply:
Done.